src/main/java/edu/harvard/iq/dataverse/pidproviders/doi/AbstractDOIProvider.java

```diff
@@ -26,6 +27,25 @@ public abstract class AbstractDOIProvider extends AbstractPidProvider {
 
     public AbstractDOIProvider(String id, String label, String providerAuthority, String providerShoulder, String identifierGenerationStyle, String datafilePidFormat, String managedList, String excludedList) {
         super(id, label, DOI_PROTOCOL, providerAuthority, providerShoulder, identifierGenerationStyle, datafilePidFormat, managedList, excludedList);
+        //Create case insensitive (converted toUpperCase) managedSet and excludedSet
+        HashSet<String> cleanManagedSet = new HashSet<String>();
+        for(String entry: managedSet) {
+            if(entry.startsWith(DOI_PROTOCOL)) {
+                cleanManagedSet.add(DOI_PROTOCOL + entry.substring(DOI_PROTOCOL.length()).toUpperCase());
+            } else {
+                logger.warning("Non-DOI found in managedSet of pidProvider id: " + getId() + ": " + entry + ". Entry is being dropped.");
+            }
+        }
+        managedSet = cleanManagedSet;
+        HashSet<String> cleanExcludedSet = new HashSet<String>();
+        for(String entry: excludedSet) {
+            if(entry.startsWith(DOI_PROTOCOL)) {
+                cleanExcludedSet.add(DOI_PROTOCOL + entry.substring(DOI_PROTOCOL.length()).toUpperCase());
+            } else {
+                logger.warning("Non-DOI found in excludedSet of pidProvider id: " + getId() + ": " + entry + ". Entry is being dropped.");
+            }
+        }
+        excludedSet = cleanExcludedSet;

```

Comment:
It might be interesting to merge from develop to see how this affects the CrossRefDOIProvider which also extends the AbstractDOIProvider.

Also, would you consider moving the 2 for loops into a single method call since they are almost identical?

Reply:
I merged and made a private clean method.